### DISCUSSION TITLE
gobang is open for questions!

### DISCUSSION THREAD

**Discussion by TaKO8Ki** [TaKO8Ki/gobang#96] [General]
I'm trying out GitHub Discussions. Please feel free to ask questions in a new discussion.

**Comment by godsbrick** [comment]
Can I get the code for a well designed and functioning calculator?

**Comment by lifesign** [comment]
cool~

**Comment by ac5tin** [comment]
i got an error while trying to connect to a postgres database: `pool timed out while waiting for an open connection`

**Reply by TaKO8Ki** [reply]
Thank you for your question! Can you create a new discussion about that?

**Reply by ac5tin** [reply]
i've created a new discussion: https://github.com/TaKO8Ki/gobang/discussions/110

**Comment by ac5tin** [comment]
the readme doesnt show that you need to put a `password` field in the config file `config.toml` but in reality it actually requires it right?

**Reply by TaKO8Ki** [reply]
The password field in config file is optional. So if your role has a password, you need to set a password field.

**Reply by ac5tin** [reply]
oh cool, okok,

**Comment by ac5tin** [comment]
does it refresh the table list automatically or i have to press a key like for example `r` ?

**Reply by TaKO8Ki** [reply]
In order to refresh the table list, you need to reselect a connection currently. I think it is inconvenient and I will enable gobang to refresh the table list and records easily in some way.

Btw, even a simple question can be helpful to others, so could you create a new discussion next time?

**Comment by ac5tin** [comment]
is there a way to sort a table by a specific column?

**Comment by friendbear** [comment]


**Reply by TaKO8Ki** [reply]
Thank you for your question! I think even simple questions can be helpful to others, so could you create a new discussion like the following?

https://github.com/TaKO8Ki/gobang/discussions/110

**Reply by friendbear** [reply]
OK!  Open #124

**Comment by ZayRTun** [comment]
Hi, I added a mysql connection in config.toml  as such: 
```
type = "mysql"
user = "root"
host = "localhost"
port = 3306
database = "project_restaurant"
```
And when I try to connect I get the following:
```
error returned from database: 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)
```
I am not using a password.. what should I do?

**Comment by marvindore** [comment]
Do you foresee Gobang adding a feature like Microsoft SQL Profiler where you can view all commands internal/external executed on the database?
